platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/userlocation/MockLocationEngine.java

```diff
@@ -1,38 +1,55 @@
 package com.mapbox.mapboxsdk.testapp.activity.userlocation;
 
-
+import android.animation.AnimatorListenerAdapter;
+import android.animation.TypeEvaluator;
+import android.animation.ValueAnimator;
 import android.location.Location;
-import android.os.Handler;
 
 import com.mapbox.services.android.telemetry.location.LocationEngine;
 import com.mapbox.services.android.telemetry.location.LocationEngineListener;
 
+import timber.log.Timber;
+
 /**
- * Sample LocationEngine that provides mocked locations simulating GPS updates
+ * Sample LocationEngine that provides mocked LOCATIONS simulating GPS updates
  */
 public class MockLocationEngine extends LocationEngine {
```

Comment:
@tobrun : @cammace is making the `MockLocationEngine` reusable across projects with https://github.com/mapbox/mapbox-java/pull/476. Could we use it here as well? If so, let's ticket a follow up ticket to make that happen.